### DISCUSSION TITLE
Open source question

### DISCUSSION THREAD

**Discussion by HeroFluxx** [Mojang/minecraft-editor#132] [Q&A] [answered]
This not related to editor mode 
But is there any possibility that you open source more things from the game 
I remember there used to be discussions about open sourcing parts of the engine 
But that never happened sadly
Also, for historical purposes could you please release the source code for older versions of Minecraft PE?

**Comment by minecraftchloe** [comment, marked as answer]
Open sourcing more things from the game (I assume you're referring to Bedrock): I won't speak on behalf of other Minecraft teams, however I can speak on behalf of the Editor team. In the context of the Editor, which is the focus of this forum, we're looking at the Scripting API and Editor API as a way for us to create an open model for supporting the creator community in building content and tools. 

Releasing source code for older versions of Minecraft PE: No. 

Please kindly keep questions, etc. in this forum focused on Editor Mode! Thank you!! 🥰

**Reply by HeroFluxx** [reply]
My apologies for any inconvenience that I caused
